gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,5 +1,5 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-6.7.1-bin.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-7.5.1-bin.zip
```

Comment:
Please update the Gradle wrapper using the following command: `./gradlew wrapper --gradle-version 7.5.1`. This will also update the script files if needed.

See the docs for more information: https://docs.gradle.org/current/userguide/gradle_wrapper.html#sec:upgrading_wrapper

Reply:
Thanks for letting me know the better way. Done.